buildSrc/src/main/java/com/uber/okbuck/core/dependency/exporter/DependencyExporterModel.java

```diff
@@ -0,0 +1,73 @@
+package com.uber.okbuck.core.dependency.exporter;
+
+import org.codehaus.plexus.util.StringUtils;
+import org.gradle.api.artifacts.ExternalDependency;
+
+import javax.annotation.Nullable;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+public class DependencyExporterModel {
+
+  @Nullable
+  private final String name;
+  @Nullable
+  private final String version;
+  @Nullable
+  private final String group;
+  private final boolean force;
+  @Nullable
+  private Set<String> excludeRules = new TreeSet<>();
+  private final boolean transitive;
+
+  public DependencyExporterModel(ExternalDependency externalDependency) {
+    name = externalDependency.getName();
+    version = externalDependency.getVersion();
+    group = externalDependency.getGroup();
+    force = externalDependency.isForce();
+    transitive = externalDependency.isTransitive();
+
+    if (externalDependency.getExcludeRules() != null) {
+      excludeRules = externalDependency.getExcludeRules().stream().flatMap(e -> {
+        Set<String> set = new TreeSet<>();
```

Comment:
seems like we will always have one entry, we could use a map instead of flatmap and string instead of set